userproject/include/generateRun.h

```diff
@@ -154,7 +155,8 @@ class GenerateRunBase
             cmd += " -c";
         }
 
-        cmd += " && make clean all";
+        const unsigned int numThreads = std::thread::hardware_concurrency();
+        cmd += " && make -j " + std::to_string(numThreads) + " clean all";
```

Comment:
interesting idea ... I doubt it will speed up massively?

Reply:
Well, now the generated code is split across multiple modules (synapse update, neuron update, init and runner), there's at least 4 decently independent jobs so it makes a reasonable difference actually